src/groupeddataframe/splitapplycombine.jl

```diff
@@ -438,8 +438,9 @@ function _combine_process_pair_astable(optional_i::Bool,
     end
     if out_col_name isa AbstractVector{Symbol}
         if length(out_col_name) != length(nms)
-            throw(ArgumentError("Number of returned columns does not " *
-                                "match the length of requested output"))
+            throw(ArgumentError("Number of returned columns is $(length(nms)) " *
+                    "and it does not match the length of requested " *
+                    "output which is $(length(out_col_name))"))
```

Comment:
```suggestion
            throw(ArgumentError("Number of returned columns is $(length(nms)) " *
                                "and it does not match the length of requested " *
                                "output which is $(length(out_col_name))"))
```

Reply:
BTW, the "length of requested output" is actually the number of output names provided, right? Maybe that would be clearer phrased like that?

Reply:
yes - I fix it in both places